examples/CustomHttpClient.php

```diff
@@ -23,11 +23,11 @@ class HttpCliImp implements HttpClientInterface {
 
     public function __construct(Guzzle $guzz)
     {
-        $this->$_guzz = $guzz;
+        $this->_guzz = $guzz;
```

Comment:
The attribute declaration on line 22 needs to be updated to `private _guzz` as well

Reply:
I believe on line 22 it should still be `private $_guzz` because it is still a variable.. no?

Reply:
Oh right right